internal/attestation/aws/validator_test.go

```diff
@@ -114,65 +118,21 @@ func TestTpmEnabled(t *testing.T) {
 	}
 }
 
-func TestValidateCVM(t *testing.T) {
-	mockDocBase := imds.InstanceIdentityDocument{
-		DevpayProductCodes:      []string{"devpayProductCodes"},
-		MarketplaceProductCodes: []string{"marketplaceProductCodes"},
-		AvailabilityZone:        "availabilityZone",
-		PrivateIP:               "privateIp",
-		Version:                 "version",
-		Region:                  "region",
-		InstanceID:              "instanceId",
-		BillingProducts:         []string{"billingProducts"},
-		InstanceType:            "instanceType",
-		AccountID:               "accountId",
-		PendingTime:             time.Now(),
-		ImageID:                 "imageId",
-		KernelID:                "kernelId",
-		RamdiskID:               "ramdiskId",
-		Architecture:            "architecture",
-	}
-
-	mockDocTpm := mockDocBase
-	mockDocTpm.ImageID = "ami-0e6b1588acc5a055f"
-	mockDocTpm.Region = "us-east-2"
-
-	mockDocNoTpm := mockDocBase
-	mockDocNoTpm.ImageID = "ami-0b59bfac6be064b78"
-	mockDocTpm.Region = "us-east-2"
-
-	attDocTpm := vtpm.AttestationDocument{}
-	attDocTpm.UserData, _ = json.Marshal(mockDocTpm)
-	attDocTpm.Attestation = &attest.Attestation{}
+type AWSMetadataStub struct {
+	describeImagesErr        error
+	describeImagesTPMSupport string
+}
 
-	attDocNoTpm := vtpm.AttestationDocument{}
-	attDocNoTpm.UserData, _ = json.Marshal(mockDocNoTpm)
-	attDocNoTpm.Attestation = &attest.Attestation{}
+// DescribeImages is a mock function for testing.
+// Although the original function works for multiple supplied image id's, this function ignores this, since in our code we also only get the information for one image.
```

Comment:
Methods on test types don't need their own doc comment, unless they do something overly complex which should be documentet, or the type is exported for use in other modules.
Other wise looks good ;)